### PR TITLE
Fixes #27537 - Show tooltip for vm memory size

### PR DESCRIPTION
**Pull request by xprazak2**

### CODE REVIEW

**Comment by ares** [comment]
this is fine but just a workaround for broken spinner field, I think @ezr-ondrej looked at the issue a while ago so he may have some lead on how to fix it properly

**Comment by ezr-ondrej** [comment]
Yeah, this will work, because different javascript is catching it, but it doesn't solve the core issue.
It's a working workaround and I believe better solution anyway :+1:

File: app/helpers/form_helper.rb [hunk 1]
```diff
@@ -220,7 +220,7 @@ def byte_size_f(f, attr, options = {})
     options[:label_help] = _("When specifying custom value, add 'MB' or 'GB' at the end. Field is not case sensitive and MB is default if unspecified.")
     options[:help_block] ||= soft_limit_warning_block
     options[:help_block] += f.hidden_field(attr, :class => "real-hidden-value", :id => nil)
-    options[:label_help_options] = { :rel => 'popover-modal' }
+    options[:label_help_options] = { :data => { :placement => 'right' } }
```
Comment (ezr-ondrej) [inline app/helpers/form_helper.rb]: Maybe we can just delete it, to be consistent with other fields whatever we decide to use globaly? Or is there a reason to be always on the right?

Comment (xprazak2) [reply app/helpers/form_helper.rb]: It's just that other tooltips in the form are on the right. But it is a default, so I deleted it as suggested.

**Review by ezr-ondrej — APPROVED** [review]
ACK :+1: Thanks @xprazak2 merging.

**Comment by ezr-ondrej** [comment]
failures of travis just an fetch failure on one version, ignoring...

**Pull request merged**